lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/builtins/MethodNodeClassGenerator.java

```diff
@@ -70,6 +70,7 @@ public void generate(
               + "\"\"\""
               + moduleOwnerInfo
               + ")");
+      out.println("@ImportStatic(" + ownerClazz.fullyQualifiedName() + ".class)");
```

Comment:
This allows us to call `Vector.createValueForIndexNode()` from the generated node class.

Reply:
Where do we use this?